src/test/java/guitests/KeyBindingsGuiTest.java

```diff
@@ -14,6 +14,11 @@
 
 /**
  * Tests key bindings through the GUI
+ * Special Note:
+ * In travis ci headlfull testing, the following were observed:
+ * 1) Keybinding tests failed as shortcuts are not triggered.
+ *    - First shortcuts get triggered as normal, second shortcuts onwards failed to get triggered.
+ * Solution: Click on the listview first before firing shortcuts. (Use personListPanel.clickOnListView())
```

Comment:
It's not clear if the solution is being used in the current code or it's just a suggestion. Also mention that it seems to work although we don't know exactly why.